swap_meet/item.py

```diff
@@ -1,2 +1,36 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0.0, age=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
```

Comment:
You could make this logic more concise here by using a ternary operator:
```suggestion
        self.id = id if id is not None else uuid.uuid4().int
```
The logic is relatively simplistic, you wouldn't lose out on much, if any, readability.